samples/cpp/image_generation/README.md

```diff
@@ -210,3 +212,43 @@ Test finish, load time: 9356.00 ms
 Warmup number:1, first generate warmup time:85008.00 ms, infer warmup time:84999.88 ms
 Generate iteration number:3, for one iteration, generate avg time: 84372.34 ms, infer avg time:84363.95 ms, all text encoders infer avg time:76.67 ms, vae encoder infer avg time:0.00 ms, vae decoder infer avg time:4470.33 ms
 ```
+
+### Run multiple generations with different prompt in parallel
```

Comment:
We can do it later, but we need to align these instructions with those for other samples. They should show how to run the multiconcurrency samples